utils/misc/misc.go

```diff
@@ -1214,7 +1214,14 @@ func GetWarehouseURL() (url string) {
 
 func GetDatabricksVersion() (version string) {
 	url := fmt.Sprintf(`%s/databricksVersion`, GetWarehouseURL())
-	resp, err := http.Get(url)
+	req, err := http.NewRequest(http.MethodGet, url, http.NoBody)
+	if err != nil {
+		return
+	}
+	client := &http.Client{
+		Timeout: config.GetDuration("HttpClient.timeout", 10, time.Second),
```

Comment:
we need to be consistent with all other declarations of default value for this config property
```suggestion
		Timeout: config.GetDuration("HttpClient.timeout", 30, time.Second),
```